### DISCUSSION TITLE
Support for Mac native notifications

### DISCUSSION THREAD

**Discussion by youngxhui** [Floorp-Projects/Floorp#1268] [Feature Request]
now, notifications in mac like this

![image](https://github.com/Floorp-Projects/Floorp/assets/16971804/b066612c-4835-4791-a7de-6797c2ed2332)

but other browser, notifiactions is better in mac.

![image](https://github.com/Floorp-Projects/Floorp/assets/16971804/6a32e9ad-b1c0-4aa2-9ed6-6b63832949b8)

testing website: https://www.bennish.net/web-notifications.html

**Comment by CutterKnife** [comment]
Set `alerts.useSystemBackend` to `true`.
<img width="370" alt="スクリーンショット 2024-07-10 18 31 06" src="https://github.com/Floorp-Projects/Floorp/assets/110194318/3660d920-bff6-4f0c-a2de-24951ea280da">

**Reply by youngxhui** [reply]
thanks

**Reply by erifneerg** [reply]
for those clueless go to about:config

this also should be just set to true by default.
